modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartClientCodegen.java

```diff
@@ -73,6 +73,10 @@
     protected String apiTestPath = "test" + File.separator;
     protected String modelTestPath = "test" + File.separator;
 
+    // Names that must not be used as model names because they clash with existing
+    // default imports (dart:io, dart:async, package:http etc.) but are not basic dataTypes.
+    protected Set<String> additionalReservedWords = new HashSet<>();
```

Comment:
Can we initialize this once in constructor? No need for a constructor there when it's in DartClientCodegen() already

Reply:
this would also make sure children of `DartClientCodegen` make sure to call `super()` which is what we want I think